api/notd/api/endpoints_v1.py

```diff
@@ -119,3 +133,8 @@ class RefreshAccountTokenOwnershipsRequest(BaseModel):
 
 class RefreshAccountTokenOwnershipsResponse(BaseModel):
     pass
+class GetCollectionActivityRequest(BaseModel):
+    pass
+
+class GetCollectionActivityResponse(BaseModel):
+    collectionActivity: List[ApiCollectionActivity]
```

Comment:
this is wierd. if its a list then the variable should be a list. see comment in api.